### ISSUE TITLE
Basic Retrieval Model

### ISSUE THREAD

**Issue by KarylReyne** [DennisKleinhans/tue-search#1]
- [x] ranker
- [x] eval metric
- [ ] save model weights to disk [link](https://pytorch.org/tutorials/beginner/saving_loading_models.html)
- [ ] plot loss/pcc for train and test
- [x] try different loss functions [link](https://vinija.ai/concepts/loss/)
- [ ] try pretrained word embeddings
- [ ] try learning rate decay/dropout/L2 regularization/noise injection
